.gitpod.yml

```diff
@@ -0,0 +1,29 @@
+image:
+    file: .gitpod.dockerfile
+ports:
+    - port: 5000
+      onOpen: open-preview
+tasks:
+    - init: >
+        cp config/database.yml.gitpod config/database.yml &&
+        cp config/config.yml.example config/config.yml &&
+        cp db/schema.rb.example db/schema.rb &&
+        bundle install --without production &&
+        yarn install &&
```

Comment:
@jywarren try this!
```suggestion
        yarn config set ignore-engines true && yarn install;
```

Reply:
OK, doing so!!

Reply:
sorry that didnt help!! 

other suggestions:
- maybe we need to be caching yarn instead of the default npm? https://docs.travis-ci.com/user/caching/#yarn-cache

- and then in package.json we have `"components-jqueryui"` and `"jqueryui"` pointing to the same package. maybe something wrong there?